src/Native/Unix/System.Security.Cryptography.Native.Apple/pal_ecc.cpp

```diff
@@ -0,0 +1,75 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include "pal_ecc.h"
+
+extern "C" int
+AppleCryptoNative_EccGenerateKey(int32_t keySizeBits, SecKeyRef* pPublicKey, SecKeyRef* pPrivateKey, int32_t* pOSStatus)
+{
+    if (pPublicKey == nullptr || pPrivateKey == nullptr || pOSStatus == nullptr)
+        return kErrorBadInput;
```

Comment:
The only supported caller is us, using `out SafeHandle` calls, which will never pass nullptr.  But if we've been doing partial input checks to maximize safety elsewhere I can follow the pattern.